omero_marshal/decode/decoders/original_file.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from .. import Decoder
+from omero.model import OriginalFileI
+
+
+class OriginalFileDecoder(Decoder):
+
+    TYPE = 'TBD#OriginalFile'
+
+    OMERO_CLASS = OriginalFileI
+
+    def decode(self, data):
+        v = super(OriginalFileDecoder, self).decode(data)
+        self.set_property(v, 'path', data.get('path'))
+        self.set_property(v, 'size', data.get('size'))
+        self.set_property(v, 'atime', data.get('atime'))
+        self.set_property(v, 'mtime', data.get('mtime'))
+        self.set_property(v, 'ctime', data.get('ctime'))
+        self.set_property(v, 'hash', data.get('hash'))
+        self.set_property(v, 'mimetype', data.get('mimetype'))
+        self.set_property(v, 'name', data.get('name'))
```

Comment:
The usefulness of one vs. the other is debatable. The only goal is to keep the dictionary keys, and case, identical to their model attribute equivalents regardless of whether their source is the OME or OMERO data model.

It could be `omero:File` as that would be in keeping with the attribute style we use elsewhere. Although, if we were doing this "properly" we would likely shoehorn the `OriginalFile` attributes into `BinaryFile`. Not sure at this point which is better. I'll have to give it some more thought.